eclipse-collections/src/main/java/org/eclipse/collections/impl/collector/SerializableDoubleSummaryStatistics.java

```diff
@@ -61,7 +63,12 @@
             max = DoubleSummaryStatistics.class.getDeclaredField("max");
             max.setAccessible(true);
         }
-        catch (SecurityException | NoSuchFieldException ignored)
+        catch (RuntimeException e)
```

Comment:
The find bugs build fails when only an Exception is caught.
http://findbugs.sourceforge.net/bugDescriptions.html#REC_CATCH_EXCEPTION